### PR TITLE
POD C | EX-116 | Add tracking events to Popups variants

### PR DESCRIPTION
**Pull request by maaartyyynaa**
https://wikia-inc.atlassian.net/browse/EX-116
cc: @Wikia/experiments-pod-c

### CODE REVIEW

File: src/ui/renderer.js [hunk 1]
```diff
@@ -353,6 +354,30 @@ export function bindBehavior( preview, behavior ) {
...
+	// preview.el.find( '.mwe-popups-buttons-section' ).click( ( event ) => {
+	preview.el.find( '.mwe-popups' ).click( ( event ) => {
+		event.stopPropagation();
+		trackExperimentsInteractions.trackPopupClick( event );
+	});
+
+	let timeoutId = null;
+	preview.el.find( '.mwe-popups' ).addEventListener(
+		'mouseover', function( event ) {
+			timeoutId = window.setTimeout(function(){
```
Comment (t-tomalak) [inline src/ui/renderer.js]: Nit: just `setTimeout` should be enough

File: src/ui/renderer.js [hunk 2]
```diff
@@ -353,6 +354,30 @@ export function bindBehavior( preview, behavior ) {
...
+				trackExperimentsInteractions.trackPopupHover( event );
+			}, 2000);
+		},
+		false
+	);
+	preview.el.find( '.mwe-popups' ).addEventListener(
+		'mouseout', function() {
+			window.clearTimeout( timeoutId )
```
Comment (t-tomalak) [inline src/ui/renderer.js]: also here window is not necessary

File: src/ui/renderer.js [hunk 3]
```diff
@@ -353,6 +354,30 @@ export function bindBehavior( preview, behavior ) {
 
 			behavior.showSettings( event );
 		} );
+
+	// Popups experiment:
+	// find the button and track click action, hover on popup
+	// preview.el.find( '.mwe-popups-buttons-section' ).click( ( event ) => {
+	preview.el.find( '.mwe-popups' ).click( ( event ) => {
```
Comment (t-tomalak) [inline src/ui/renderer.js]: Are we sure that this tracks all the clicks? Also question is if we should track all the clicks inside or only those which are on elements that "redirects"?

**Review by t-tomalak — COMMENTED** [review]
Few comments

**Comment by t-tomalak** [comment]
Also I don't see `resources/dist/index.js` file so I assume this will not fire on production (Instruction on the channel)

File: src/ui/renderer.js (hunk 3 above)
Comment (maaartyyynaa) [reply src/ui/renderer.js]: it's still in progress and I was planning to test it then update if needed

File: src/ui/renderer.js [hunk 4]
```diff
@@ -353,6 +354,29 @@ export function bindBehavior( preview, behavior ) {
 
 			behavior.showSettings( event );
 		} );
+
+	// Popups experiment:
+	// find the button and track click action, hover on popup
+	preview.el.find( '.mwe-popups-buttons-section .mwe-popups-discreet .mwe-popups-extract .').click( ( event ) => {
```
Comment (maaartyyynaa) [inline src/ui/renderer.js]:
```suggestion
	preview.el.find( '.mwe-popups-buttons-section .mwe-popups-discreet .mwe-popups-extract').click( ( event ) => {
```

**Review by latata — APPROVED** [review]

**Pull request merged**